jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/SessionManager.java

```diff
@@ -187,16 +187,15 @@ public void renewId(HttpServletRequest request) {
       ((SessionManager)getSessionManager()).callSessionIdListeners(this, oldId);
     }
     
-   
-
     public void save() {
       save(false);
     }
-    
 
     protected void save (boolean force)
     {
-      logger.fine("Session " + getId() + "is"+(dirty?" dirty":" not dirty")+(force || dirty ? " saving":" not saving"));
+      if (logger.isLoggable(Level.FINE)) {
+        logger.fine(String.format("Session %s dirty=%b force=%b", getId(),dirty,force));
```

Comment:
I see some more formatting issues, like the spaces after a comma.
Will hold any formatting related comments until you run the formatter on the code.